composer.json

```diff
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": ">=5.6",
+        "ext-curl": "*",
```

Comment:
I believe these requirements are not needed in cli mode. Would this be breaking or would it help troubleshoot for those using non-cli mode?

Reply:
It's not breaking or a big thing, but it's useful and declares which extensions are expected in order to execute functionality. During `composer install`, Composer verifies required extensions are available:
```
(Path to repository)>composer install
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested PHP extension ext-curl * is missing from your system. Install or enable PHP's curl extension.
```
In this case I've commented the `extension=curl` in php.ini (Windows user) to trigger this error. In sendgrid/sendgrid-php, all required extensions are listed - which is good.

If deploying in a cloud app (like Heroku), extensions will be installed/enabled according to composer.json. So that's why I 'suggest' to include it.